<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Entity Exporter</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="ship_entity_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="listFilter">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="entityTree">
        <property name="styleSheet">
         <string notr="true">QTreeWidget::indicator:unchecked { 
	border: 1px solid gray;
} </string>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="options_frame">
     <property name="minimumSize">
      <size>
       <width>225</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectAllButton">
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deselectAllButton">
        <property name="text">
         <string>Deselect All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="toolTip">
         <string>SCDV will automatically convert CryXml binary content to standard XML or JSON formatted files.</string>
        </property>
        <property name="title">
         <string>CryXml Conversion Format</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QComboBox" name="opt_cryxmlFmt">
           <item>
            <property name="text">
             <string>xml</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>json</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Image Conversion Format</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="opt_imgFmt">
           <item>
            <property name="text">
             <string>png</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tif</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tga</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="opt_autoUnsplitTextures">
           <property name="toolTip">
            <string>Automatically un-split &quot;dds.n&quot; files into a single &quot;dds&quot; file</string>
           </property>
           <property name="text">
            <string>Automatically unsplit textures</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="opt_autoConvertTextures">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Automatically convert DDS textures to TIF formats. This requires automatically unsplitting textures, and the original DDS file will also be extracted.</string>
           </property>
           <property name="text">
            <string>Automatically convert textures</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="opt_autoConvertSounds">
           <property name="toolTip">
            <string>Automatically convert referenced ship sound files to ogg files. The converted files will be named by their trigger name.</string>
           </property>
           <property name="text">
            <string>Automatically convert sounds</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="opt_autoConvertModels">
           <property name="toolTip">
            <string>Automatically convert CryEngine (.cga/.cgf/.skin/.chr) models to standard formats.</string>
           </property>
           <property name="text">
            <string>Automatically convert models</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="opt_createSubFolder">
           <property name="toolTip">
            <string>This will create a new folder with the entity's name in the selected export directory and extract all of the entity's data there. Otherwise, all entities will be extracted into the same selected export directory.</string>
           </property>
           <property name="text">
            <string>Create sub-folder for entity</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="opt_genModelLog">
           <property name="toolTip">
            <string>Create an addition log file containing all of the extracted model files (cga/cgf/chr/skin)</string>
           </property>
           <property name="text">
            <string>Generate Extracted Models Log</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
